llvm/test/CodeGen/NVPTX/globals_init.ll

```diff
@@ -26,6 +26,6 @@
 @Gblf64 = global [2 x double] [double 5.75e-25, double 12.25e+56]
 
 ; Make sure we fill in alignment gaps correctly.
-; CHECK-DAG: .b8 GblU[12] = {7, 6, 0, 0, 5, 4, 3, 2, 1, 0, 0, 0};
+; CHECK-DAG: .b8 GblU[12] = {7, 6, 0, 0, 5, 4, 3, 2, 1};
```

Comment:
Please add few more cases:
- all-zeros,
- all-non-zeros
- initializer with a symbol

Also, please check with old versions of ptxas (10.2 is probably the oldest version we may still care about).

Reply:
All added in `llvm/test/CodeGen/NVPTX/global-incomplete-init.ll`. I've verified this works with 10.2 `ptxas`.